### ISSUE TITLE
Thermistor Beta Constant

### ISSUE THREAD

**Issue by caternuson** [adafruit/Adafruit_CircuitPlayground#15]
**TLDR:** Thermistor Beta constant should be 3380.

- Arduino board:  Circuit Playground
- Arduino IDE version (found in Arduino -> About Arduino menu):  1.6.11
- List the steps to reproduce the problem below (if possible attach a sketch or
  copy the sketch code in too): Use existing function at a temperature other than 25 deg C.

Ref. this forum thread:
https://forums.ahttps://forums.adafruit.com/viewtopic.php?f=58&t=105254dafruit.com/viewtopic.php?f=58&t=105254

Ref. this datasheet for the Murata NTC NCP15XH103F03RC:
http://www.murata.com/~/media/webrenewal/support/library/catalog/products/thermistor/ntc/r44e.ashx

Datasheet specifies a beta constant of 3380. Current library #defines it to be 3950. The datasheet linked above provides a table of actual resistance vs. temperature values on page 15. Using that, can compare results for Steinhart approximation with different beta constants. As can be seen in the figure below, 3380 provides better results.

![fig](https://cloud.githubusercontent.com/assets/8755041/19691066/bce2a57e-9a87-11e6-9bb2-839bbcbba88f.jpg)

**Comment by ladyada** [comment]
d0h! thats right, was from old breadboard-prototyping code. fixd & bumped
